src/vs/base/browser/dom.ts

```diff
@@ -1817,6 +1817,7 @@ export const basicMarkupHtmlTags = Object.freeze([
 	'hr',
 	'i',
 	'img',
+	'input',
```

Comment:
@mjbvz, I just noticed that this array is also used here:

https://github.com/microsoft/vscode/blob/3f5961e21f7bf5b5d4aad64a65417bc3590a77af/src/vs/workbench/contrib/markdown/browser/markdownDocumentRenderer.ts#L167-L171

Is this still ok?